### DISCUSSION TITLE
Headings only recognized if there are space after #

### DISCUSSION THREAD

**Discussion by jpedroschmitz** [mdx-editor/editor#143] [Q&A]
I noticed that the headings are only correctly rendered if there is space after the `#` token.

<img width="257" alt="CleanShot 2023-10-25 at 15 04 01@2x" src="https://github.com/mdx-editor/editor/assets/26466516/a176917b-ffc4-4c32-9962-e6e50756f754">

<img width="257" alt="CleanShot 2023-10-25 at 15 07 29@2x" src="https://github.com/mdx-editor/editor/assets/26466516/dcc68221-0612-4efd-b26b-1cc09dde840d">

Is it possible to make it render the title without having to add a space?

**Comment by petyosi** [comment]
I think this is not possible, but I might be wrong. The markdown parsing comes from the https://unifiedjs.com/ tool family. Apparently, [spacing is recommended](https://www.markdownguide.org/basic-syntax/#heading-best-practices). However, if you find a way to make the unified tooling recognize headings without space, let me know how. 

If you have an existing markdown content, I would recommend is to process your markdown with a regex to add the spacing - should be fairly simple.
